public/app/types/unified-alerting-dto.ts

```diff
@@ -122,6 +122,7 @@ interface PromRuleDTOBase {
   evaluationTime?: number;
   lastEvaluation?: string;
   lastError?: string;
+  uid?: string;
```

Comment:
a bit confused about this change; do Prometheus rules now also have an optional UID? If this is only for Grafana-managed can we move it to the more specific type?